package.json

```diff
@@ -3,31 +3,51 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./scripts/www",
-    "debug": "cross-env DEBUG=bbsite nodemon ./scripts/www",
-    "lint": "eslint --ext .js --ext .jsx src test templates scripts app.js gulpfile.js",
+    "copy-bootstrap": "cp -R node_modules/bootstrap/less/ src/client/stylesheets/bootstrap",
```

Comment:
A lot of these are pretty gnarly. It would be very helpful to break them out into sh scripts or the like to make them easier to maintain.